.github/workflows/release.yml

```diff
@@ -38,7 +38,7 @@ jobs:
           passphrase: ${{ secrets.PASSPHRASE }}
 
       - name: Run GoReleaser
-        uses: goreleaser/goreleaser-action@v5
+        uses: goreleaser/goreleaser-action@v6
         with:
           version: latest
```

Comment:
Tests are unhappy. It seems some modification is required in the workflow file

Reply:
I think that setting specific version like `version: '~> v2'` or removing it should do the trick according to the docs: https://github.com/goreleaser/goreleaser-action?tab=readme-ov-file#inputs

Reply:
Yeah, I'm trying to figure what's the way forward. I don't we can drop the `.goreleaser.yaml` file altogether or just need update it with a version.

Reply:
We need the goreleaser for creating release binaries, so fixing the version should suffice

Reply:
It was the schema - fixed in https://github.com/terraform-lxd/terraform-provider-lxd/pull/480. Thanks